rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -0,0 +1,67 @@
+from rj_msgs import msg
+import stp.rc as rc
+import numpy as np
+
+from typing import List
+
+def robotstate_to_robot(robot_msg: msg._robot_state.RobotState, index: int) -> rc.Robot:
```

Comment:
This will have to be changed with the rc.py rewrite but I can do that after this is merged and before the rc.py changes are merged in.

Reply:
With the expanded rc.py this constructor will also take in a RobotStatus message. Not sure if I'm forgetting anything else.

(That's relevant to @liquidmetal9015, but @HussainGynai doesn't have to care about it for this PR.)